libs/application/templates/financial-aid/src/fields/Confirmation/Confirmation.tsx

```diff
@@ -33,16 +33,22 @@ const Confirmation = ({ application, refetch }: FAFieldBaseProps) => {
       <Box marginTop={2}>
         <DescriptionText text={confirmation.nextSteps.content} />
       </Box>
-      <Text as="h3" variant="h3" marginTop={[4, 4, 5]}>
-        {formatMessage(confirmation.sharedLink.title)}
-      </Text>
-      <Box marginTop={2}>
-        <CopyUrl
-          inputLabel={formatMessage(copyUrl.inputLabel)}
-          buttonLabel={formatMessage(copyUrl.buttonLabel)}
-          successMessage={formatMessage(copyUrl.successMessage)}
-        />
-      </Box>
+
+      {externalData.nationalRegistry?.data?.applicant?.spouse && (
```

Comment:
Don't we need to add to this check if the applicant does not have a spouse in Þjóðskrá but still checked that he has unregistered spouse? Similar to the hasSpouse function here: libs/application/templates/financial-aid/src/lib/utils.ts